vms/platformvm/warp/payload/README.md

```diff
@@ -0,0 +1,44 @@
+# Payload
+
+An Avalanche Unsigned Warp Message already includes a `networkID`, `sourceChainID`, and `payload` field. The `payload` field is parsed into one of the types included in this package to be handled by the EVM.
+
+## AddressedPayload
+
+AddressedPayload:
+```
++---------------------+----------+-------------------+
+|             codecID :   uint16 |           2 bytes |
++---------------------+----------+-------------------+
+|              typeID :   uint32 |           4 bytes |
++---------------------+----------+-------------------+
+|       sourceAddress : [20]byte |          20 bytes |
++---------------------+----------+-------------------+
+|             payload :   []byte |  4 + len(payload) |
++---------------------+----------+-------------------+
+                                 | 30  + len(payload) |
+                                 +-------------------+
+```
+
+- `codecID` is the codec version used to serialize the payload and is hardcoded to `0x0000`
+- `typeID` is the payload type identifier and is `0x00000000` for `AddressedPayload`
+- `sourceAddress` is the address that called `sendWarpPrecompile` on the source chain
```

Comment:
I think we should make this more generic. Seems a bit arbitrary to force 20 byte addresses for all VMs. (or assume precompile language here).
(Actually seems the code is updated, should we update this document/diagram to match?)